mlir/lib/Quantum/Utils/RemoveQuantum.cpp

```diff
@@ -19,8 +19,9 @@
 #include "Quantum/IR/QuantumInterfaces.h"
 #include "Quantum/IR/QuantumOps.h"
 #include "Quantum/Utils/RemoveQuantum.h"
+#include "mlir/Dialect/Arith/IR/Arith.h"
+#include "mlir/Dialect/Tensor/IR/Tensor.h"
 #include "mlir/IR/PatternMatch.h"
-
 using namespace mlir;
```

Comment:
The formatting here is off now (local and third-party imports were merged into one block, using directive in the same block as the includes). Please move combine the mlir includes with the existing llvm one, and add the separations back, should look something like this:
```
#include "llvm/ADT/SmallPtrSet.h"
#include "mlir/Dialect/Arith/IR/Arith.h"
#include "mlir/Dialect/Tensor/IR/Tensor.h"
#include "mlir/IR/PatternMatch.h"

#include "Quantum/IR/QuantumInterfaces.h"
#include "Quantum/IR/QuantumOps.h"
#include "Quantum/Utils/RemoveQuantum.h"

using namespace mlir;
```

Reply:
Done :)